src/azul/__init__.py

```diff
@@ -1566,8 +1566,13 @@ def docker_image_manifests_path(self) -> Path:
 
     allowed_v4_ips_term = 'allowed_v4_ips'
 
+    # We use two rate rules, one to block requests at a given threshold, and one
+    # to trigger an alarm at a higher threshold.
+
     waf_rate_rule_name = 'RateRule'
```

Comment:
Do consider that the comment format appears inconsistent with our style.
```suggestion
    # We use two rate rules, one to block requests exceeding a set threshold,
    # and another to triggering an alarm at a higher threshold.
    #
    waf_rate_rule_name = 'RateRule'
```